### PR TITLE
[Doc] [NextJS] Revise documentation for deploying-to-production/export

### PR DESCRIPTION
**Pull request by AncaIO**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Documentation update (non-breaking change; modified files are limited to the `/docs` directory)

## Checklist:

- [ ] I have read the Contributing guide.
- [ ] My code follows the code style of this project.
- [ ] My code/comments/docs fully adhere to the Code of Conduct.
- [ ] My change is a code change and it requires an update to the documentation.
- [ ] My change is a documentation change and it requires an update to the navigation.

### CODE REVIEW

File: docs/data/routes/docs/nextjs/deploying-to-production/export/en.md [hunk 1]
```diff
@@ -3,4 +3,76 @@ name: export
...
+You can serve your JSS Next.js application as static HTML files using any static hosting service or content delivery network. 
+
+> Static HTML export does not support: 
+>
+> * Internationalized routing.
+> * Request rewrites.
+> * Visitor identification. 
+> * The default image `loader` used by the `next/image` component.
+>
+> See the  Next.js  [Static HTML Export Caveats](https://nextjs.org/docs/advanced-features/static-html-export/) documentation for more information.
```
Comment (nickwesselman) [inline docs/data/routes/docs/nextjs/deploying-to-production/export/en.md:17]: Should we do a header deep link here?

https://nextjs.org/docs/advanced-features/static-html-export#caveats

File: docs/data/routes/docs/nextjs/deploying-to-production/export/en.md [hunk 2]
```diff
@@ -3,4 +3,76 @@ name: export
...
+
+To export your JSS Next.js application as static HTML files, you must: 
+
+1. [Prepare your application for static HTML export]().
```
Comment (nickwesselman) [inline docs/data/routes/docs/nextjs/deploying-to-production/export/en.md:21]: Do we need header links here?

**Review by nickwesselman — APPROVED** [review]

File: docs/data/routes/docs/nextjs/deploying-to-production/export/en.md (hunk 1 above)
Comment (AncaIO) [reply docs/data/routes/docs/nextjs/deploying-to-production/export/en.md:17]: Yes. Will do.

File: docs/data/routes/docs/nextjs/deploying-to-production/export/en.md (hunk 2 above)
Comment (AncaIO) [reply docs/data/routes/docs/nextjs/deploying-to-production/export/en.md:21]: Hmmm, they probably make little sense. Especially if someone jumps straight to export :D Will fix.

File: docs/data/routes/docs/nextjs/deploying-to-production/export/en.md [hunk 3]
```diff
@@ -3,4 +3,76 @@ name: export
 routeTemplate: ./data/component-templates/article.yml
 title: Using `next export` with the Next.js sample app
 ---
-# Walkthrough: Using `next export` with the Next.js sample app
\ No newline at end of file
+# Walkthrough: Using `next export` with the Next.js sample app
```
Comment (nickwesselman) [inline docs/data/routes/docs/nextjs/deploying-to-production/export/en.md:6]: Just realized we should say something here about the fact that the Sitecore Media Library is not exported, and that a running Content Delivery instance is still needed for the media library.

**Review by nickwesselman — CHANGES_REQUESTED** [review]
We should add something on media, otherwise good!

**Review by nickwesselman — APPROVED** [review]

**Pull request merged**